snippets.json

```diff
@@ -0,0 +1,100 @@
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Festival Lighting",
+    "codepen-link": "https://codepen.io/pen/NWOzdvK",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/477cc242-4b77-4e50-bcd7-d64cfce2ed34",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Festival%20Lighting",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Indian Flag",
+    "codepen-link": "https://codepen.io/pen/xxyzqVa",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/68843a36-a400-4e48-8907-435e68d8cdc0",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Indian%20Flag",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Image Carousel I",
+    "codepen-link": "https://codepen.io/pen/gOBKRNV",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/5fc52924-7403-4178-a7ab-fc796cfab567",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Carousel%20I",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Image Carousel II",
+    "codepen-link": "https://codepen.io/pen/bGmjbVo",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/a8ef65b5-873c-4365-b4be-485bae463e87",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Carousel%20II",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Color Wheel",
+    "codepen-link": "https://codepen.io/pen/GRYXvKw",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/b4a4bc20-d43b-4bc0-b86d-ae89ae84ff42",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Color%20Wheel",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Video Player",
+    "codepen-link": "https://codepen.io/ahmedelmsery/pen/zYmmxNw",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/4fa44080-b6cf-4a5a-b0eb-53012b1e39a7",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Video%20Player",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Bar Chart",
+    "codepen-link": "https://codepen.io/ahmedelmsery/pen/vYVQGLP",
+    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/700f6a4f-b376-42cd-89fb-d40ace55acea",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Chart/BarChart",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Loading Page",
+    "codepen-link": "https://codepen.io/ahmedelmsery/pen/zZVLMN",
+    "image-link": "https://res.cloudinary.com/duwcigilo/image/upload/v1684336253/Loading_image_ig5hh7.jpg",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Loading%20Page",
+    "author": "Deveesh-Shetty"
+  },
```

Comment:
The author for this snippet is someone else check the codepen link and author name from #8

Reply:
Changes have been made, please review again.